kernel/src/kernel.rs

```diff
@@ -877,15 +876,17 @@ impl Kernel {
                             subscribe_num: subdriver_number,
                         };
 
+                        // TODO: when the compiler supports capability types bring this back
+                        // as a NonNull type.
```

Comment:
Is there an upstream tracking issue or similar we can add to this comment?

Reply:
Used the main CHERI one